test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
```

Comment:
It'd be great to wrap all the tests in a top level `test.describe( 'Navigation block', () => {} )` description, so that the console generates the right output.